contrib/admin/rm-whitespace.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
Sure, but this is above removing trailing whitespace, as opposed to collapsing redundant whitespace, so can the name be more specific?

Reply:
Modified!